components/Player/Pages/Overview.jsx

```diff
@@ -24,7 +24,7 @@ const PlayerMatchesTable = createTable(
 );
 const PlayerHeroesTable = createTable(
   playerHeroes.getPlayerHeroesById,
-  (state, sortState, playerId) => (sortState ? sortPlayerHeroes(playerId)(state) : transformPlayerHeroesById(playerId)(state)),
+  (state, sortState, playerId) => (sortState ? sortPlayerHeroes(playerId, 20)(state) : transformPlayerHeroesById(playerId, 20)(state)),
```

Comment:
It doesn't seem very useful for tables with limited number of rows displayed. Thid I'd only giving us the ability for this kind of behavior anyways. Once we add pagination we don't need to use this, but until then it feels weird to do the full data set (for example on heroes) when you sort and it's rows that you normally can't get to.